product_docs/docs/biganimal/release/overview/updates.mdx

```diff
@@ -8,6 +8,9 @@ EDB reserves the right to upgrade customers to the latest minor version without
 
 In some cases these updates might terminate existing network connections to your clusters. If that happens, the outage is typically less than 30 seconds. Be sure your applications are configured to automatically reconnect when connections are interrupted. Most modern database libraries do this by default. 
 
+If you want to control when the updates are pushed to your clusters, you can specify a weekly maintenance window. BigAnimal displays a *scheduled maintenance* message on your cluster list four hours prior to the scheduled maintenance time to remind you to prepare for the upcoming maintenance window. This allows you to make any necessary preparations, such as saving your work and closing any open connections. For more information on specifying maintenance windows, see [Maintenance](/biganimal/latest/getting_started/creating_a_cluster/#maintenance). 
```

Comment:
can we change "to remind you to prepare for" to "to remind you of"? Since the preparation is detailed in the next sentence I don't think we need to mention it twice

Reply:
Done